cmd/deploy_test.go

```diff
@@ -1018,3 +1019,60 @@ func TestDeploy_NamespaceRedeployWarning(t *testing.T) {
 		t.Fatalf("expected function to be updated with namespace 'myns'.  got '%v'", f.Namespace)
 	}
 }
+
+// TestDeploy_NamespaceRedeployNoWarning ensures that the --remote flag is
+// persisted in the global config.
+func TestDeploy_persistRemoteFlag(t *testing.T) {
+	root, rm := Mktemp(t)
+	defer rm()
+
+	// A function with minimum required values for deployment
+	f := fn.Function{
+		Root:     root,
+		Name:     "myfunc",
+		Runtime:  "go",
+		Registry: "example.com/alice",
```

Comment:
In this case, we should actually be able to omit everything except `root` and `runtime` (all those noop implementations don't need a name or registry)